### PR TITLE
Added basic styles and components

### PR DESCRIPTION
**Pull request by shrihari**
Added basic styles and components to build on top of.

image

### CODE REVIEW

**Comment by tjvc** [comment]
Looks like you need to rebase this onto master @shrihari. You have a commit from Gonçalo on your branch that we removed when we rewrote the history before making this repo public - apologies for not making you aware of this!

File: src/scss/custom.scss [hunk 1]
```diff
@@ -4,6 +4,41 @@
...
+// Buttons + Forms
+
+$input-btn-border-width: 2px;
+
+
+// Buttons
 
-$body-color: $cardinal-red;
+// $btn-padding-y: .75rem;
+// $btn-font-size: .75rem;
```
Comment (tjvc) [inline src/scss/custom.scss]: Do we need these?

File: src/stories/02-components.js [hunk 2]
```diff
@@ -2,15 +2,28 @@ import { storiesOf } from '@storybook/html';
...
+  `)
+  .add('Button - secondary', () => `
+    <button type="button" class="btn btn-secondary">Cancel</button>
+  `)
+  .add('Button - with icon', () => `
+    <button type="button" class="btn btn-success"><i class="material-icons">bookmark</i> Save</button>
+  `)
+  .add('Text input - with label', () => `
+    <label for="exampleInputEmail1">Email address</label>
+    <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter email">
```
Comment (tjvc) [inline src/stories/02-components.js]: Should these be inside a `form-group`?

File: src/scss/app.scss [hunk 3]
```diff
@@ -3,3 +3,15 @@
...
+
+.material-icons {
+  font-size: 1em;
+  vertical-align: middle;
+  margin-top: -0.25em;
+}
+
+.custom-select {
+	-webkit-appearance: none;
+	-moz-appearance: none;
```
[the remark this answers is not in the document]
Comment (shrihari) [reply src/scss/app.scss]: It's probably because I copy-pasted these two properties. I'll change it. 👍

[the remark this answers is not in the document]
Comment (shrihari) [reply src/scss/app.scss]: I checked about this. I can confirm that the browsers are defaulting to a different value, at least for the `<select>`s.

However, Bootstrap already contains `appearance: none` (without vendor prefixes) in the source code. Their [demo page](https://getbootstrap.com/docs/4.3/components/forms/#select-menu) includes the vendor prefixes, but I wasn't able to get them to work on Bridget. So I've added these two styles for now. Will look into fixing the issue and removing these two styles.

File: src/scss/custom.scss (hunk 1 above)
Comment (shrihari) [reply src/scss/custom.scss]: I was going to work on these next. I'll remove them now.

File: src/stories/02-components.js (hunk 2 above)
Comment (shrihari) [reply src/stories/02-components.js]: Good spot, thanks! 👍 I'll update these.

File: src/scss/custom.scss [hunk 4]
```diff
@@ -4,6 +4,35 @@
 // Bootstrap is defined through a huge number of variables. You can check them
 // all at https://github.com/twbs/bootstrap/blob/v4-dev/scss/_variables.scss
 
+@import url('https://fonts.googleapis.com/css?family=Lato');
```
Comment (abenitoc) [inline src/scss/custom.scss]: Do we want to have it served from a CDN from google APIs?
Maybe problems with China?

**Pull request merged**